IncBackups/IncBackupsControl.py

```diff
@@ -738,44 +728,42 @@ def metaBackup(self):
             return 0
 
     def initiateRepo(self):
```

Comment:
changed this whole mechanism.

First it does a check for the restic repo via checking for existing snapshots, and then only proceeds to create one if it doesn't exist. This was done per the instructions in their documentation: https://restic.readthedocs.io/en/latest/075_scripting.html

But I now realize that perhaps this isn't necessary and is more efficient to just try to init rather than check snapshots. The problem was that it was throwing errors because your error checking mechanism wasn't catching the message properly.